Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "OutlineView",
     platforms: [
-      .macOS(.v11)
+      .macOS(.v10_15)
     ],
```

Comment:
It may be desirable to remove the platform restrictions, and use availability annotations on the rest of the types instead. If you want I can make that change...

```suggestion +1-0
```